sdk/attestation/attestation/test/utils/utf8.browser.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/// <reference lib="dom" />
```

Comment:
All all the `.browser` files allowed to reference the DOM types? I know in other contexts, the SDKs have to manually forward-declare runtime values.

Reply:
cc @witemple-msft

Reply:
Honestly? I am 100% monkey typing here, trying to figure out something that works (monkey typing: If an infinite number of monkeys were placed in front of a an infinite number of typewriters, eventually one of them will figure out how to use TextEncoder).

Reply:
Here's how I did it while just prototyping some stuff on our end

https://github.com/microsoft/TypeScript/pull/43987/files#diff-1d5d5e94c50d92fa606aa89bd24e1e02ab4d770535bb5827b4289a0500ee995eR7

```ts
declare let TextDecoder: undefined | (new() => { decode(buffer: ArrayBuffer | ArrayBufferView): string });
```

Then you can do the same sort of

```ts
const textDecoder = TextDecoder;
export { textDecoder as TextDecoder };
```

thing if you need.